src/telegraf.ts

```diff
@@ -96,14 +97,15 @@ class Telegraf<TContext extends Context = Context> extends Composer<TContext> {
 
   handleError: (err: any, ctx: TContext) => void = (err) => {
     console.error()
-    console.error((err.stack || err.toString()).replace(/^/gm, '  '))
+    console.error(
+      ('stack' in err ? err.stack : err.toString()).replace(/^/gm, '  ')
+    )
```

Comment:
Nothing wrong with this, but it also doesn't seem to fix anything.

Reply:
It checks that err.stack is a property on err, instead of checking if it's nullish